### PR TITLE
Deprecate elevation enum in azure-maps-render package

### PR DESCRIPTION
**Pull request by dubiety**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [v] **The pull request does not introduce [breaking changes]**
- [v] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [v] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [v] Title of the pull request is clear and informative.
- [v] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by dubiety** [comment]
@tjprescott  and @annatisch , can you help to review this PR? Thank you so much!

**Review by johanste — APPROVED** [review]

File: sdk/maps/azure-maps-render/azure/maps/render/_generated/aio/_configuration.py [hunk 1]
```diff
@@ -61,6 +57,4 @@ def _configure(self, **kwargs: Any) -> None:
         self.redirect_policy = kwargs.get("redirect_policy") or policies.AsyncRedirectPolicy(**kwargs)
         self.authentication_policy = kwargs.get("authentication_policy")
         if self.credential and not self.authentication_policy:
-            self.authentication_policy = policies.AsyncBearerTokenCredentialPolicy(
-                self.credential, *self.credential_scopes, **kwargs
-            )
+            self.authentication_policy = policies.AzureKeyCredentialPolicy(self.credential, "SAS Token", **kwargs)
```
[the remark this answers is not in the document]
Comment (annatisch) [reply sdk/maps/azure-maps-render/azure/maps/render/_generated/aio/_configuration.py:60]: Looking at your `base_client`, this should actually be `"subscription-key"`?
It looks like currently the base client is passing in its own authentication policy, however you shouldn't need to do that, you could just have this generate the correct header name.

File: sdk/maps/azure-maps-render/swagger/README.md [hunk 2]
```diff
@@ -30,12 +30,9 @@ license-header: MICROSOFT_MIT_NO_VERSION
...
 clear-output-folder: true
 python: true
-no-async: false
-add-credential: false
 title: MapsRenderClient
 disable-async-iterators: true
 python-sdks-folder: $(python-sdks-folder)
 python3-only: true
-version-tolerant: true
 models-mode: msrest
```
[the remark this answers is not in the document]
Comment (dubiety) [reply sdk/maps/azure-maps-render/swagger/README.md:37]: Hi @annatisch , definitely! We plan to overhaul it to model-less DPG before GA, but haven't had time to do it yet :)

File: sdk/maps/azure-maps-render/azure/maps/render/_generated/aio/_configuration.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dubiety) [reply sdk/maps/azure-maps-render/azure/maps/render/_generated/aio/_configuration.py]: Nice catch! I've updated the generated files. :)

**Review by annatisch — APPROVED** [review]
Thanks! :)

**Pull request merged**